src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,4 +1,57 @@
 databaseChangeLog:
   -  changeSet:
        id:  adicionar suporte ao liquibase
-       author:  izumizawa
\ No newline at end of file
+       author:  izumizawa
+
+  - changeSet:
+      id: add-address-table
+      author: Gabriel Seabra
+      changes:
+        - createTable:
+            tableName: address
+            columns:
+              - column:
+                  name: address_id
+                  type: uuid
+                  constraints:
+                    primaryKey: true
+                    primaryKeyName: address_pkey
+                    nullable: false
```

Comment:
PrimaryKey is not nullable as default. I think it's not necessary specify the nullable constraint again.